### PR TITLE
Add info on exchanging foreign driver's licenses

### LINKED ISSUE #7: Add info/links on exchanging the driver's license, maybe personal experiences for certain countries
https://www.expatica.com/ch/living/transportation/exchanging-a-foreign-drivers-licence-in-switzerland-102892/

### PR DESCRIPTION
**Pull request by shardulc**
Partially fixes #7.

### CODE REVIEW

File: _life/transportation.md [hunk 1]
```diff
@@ -58,3 +58,25 @@ If you want to come to EPFL by car, parking rules can be found [here](https://pa
...
+* You have 12 months from the date of your arrival in Switzerland to exchange your license.
+* [Depending on the country that issued your license](https://www.vd.ch/themes/mobilite/automobile/permis-de-conduire/echange-permis-etrangers/pays-ayant-une-convention-de-reconnaissance-mutuelle-avec-la-suisse), you may have no additional testing requirements (e.g. for EU countries), only the theoretical test, or both the theoretical and the practical driving test.
+  + USA is on the “only theoretical” list—but this is only because a standard US license (or to be precise, standard licenses from most US states) lets you drive some kinds of light trucks that are not covered by a standard Swiss license.
+    If you are willing to give up that extra privilege, you do not have any testing requirements.
+    This information is not available on the official website but was provided verbally by a staff member so you just have to fill out the form and trust the process.
+    I can confirm through personal experience that this worked for a WA driver’s license in 2023.
```
Comment (SolalPirelli) [inline _life/transportation.md:70]: can we avoid first person sentences, just to make it clear who folks should contact if they have questions?

File: _life/transportation.md [hunk 2]
```diff
@@ -58,3 +58,25 @@ If you want to come to EPFL by car, parking rules can be found [here](https://pa
...
+* A section of the form is to be filled out by an optician after an eye exam.
+  You can have it done at many eyeglasses shops.
+  It has a standard price (15 CHF in 2023) and takes just a few minutes.
+
+You may be used to ridiculously long wait times for in-person visits to the motor vehicles office in your home country.
+Switzerland, of course, is not like that.
```
Comment (SolalPirelli) [inline _life/transportation.md:76]: :D

**Comment by SolalPirelli** [comment]
Perhaps this should be linked somewhere from the "first year" section, since it's something that has to be done in the first 12 months?

File: _life/transportation.md [hunk 3]
```diff
@@ -58,3 +58,25 @@ If you want to come to EPFL by car, parking rules can be found [here](https://pa
...
+When I visited the Lausanne office on a weekday morning, I had to wait for barely ten minutes, and was out the door in another ten.
+
+Exchanging your foreign license typically means that you surrender your old license and get a new Swiss one.
+However, the authorities will only take your old license if they have an agreement with the issuing country or some other means to actually cancel it.
+Absent that, there is no point, so you get to keep it.
+This is true at least for US licenses.
```
Comment (SolalPirelli) [inline _life/transportation.md:82]: Maybe add something like "If you exchanged a license from another country and have some useful info, please let us know" ?

**Comment by SolalPirelli** [comment]
cc @vinitra @cancebeci @yugeshk if y'all have info about exchanging your licenses

**Comment by shardulc** [comment]
> Perhaps this should be linked somewhere from the "first year" section, since it's something that has to be done in the first 12 months?

I agree, but also it seems out of place on the current "first year" page, because all the other administrative and academic tasks are much more important than the driver's license. Any ideas?

Maybe a "Time limits" subheading on that page, with two bullet points that I can think of for now, exchanging DL within 12 months and getting health insurance within 3?

**Comment by SolalPirelli** [comment]
> Maybe a "Time limits" subheading on that page, with two bullet points that I can think of for now, exchanging DL within 12 months and getting health insurance within 3?

Sounds great. Also announcing oneself to the commune but I guess that one is hard to miss anyway?

**Comment by shardulc** [comment]
Good to merge?

**Pull request merged**